elasticsearch/kibana/init.sls

```diff
@@ -79,6 +79,34 @@ remove_default_nginx_config:
   file.absent:
     - name: {{ kibana.nginx_site_path }}/default
 
+{% if salt.grains.get('init') == 'systemd' %}
+add_node_environment_variables:
+  file.managed:
+    - name: /etc/systemd/system/kibana.service.d/kibana_env.conf
+    - makedirs: True
+    - contents: |
+        [Service]
+        {% for env in kibana.kibana_env %}
+        Environment='{{ env }}'
+        {% endfor %}
```

Comment:
Yeah, this is just to add the environment variable(s) to the unit file which is installed by default from the kibana package.